docs/troubleshooting.md

```diff
@@ -68,6 +68,37 @@ The Kubernetes team hangs out on IRC at [`#google-containers`](https://botbot.me
 
 The Kubernetes mailing list is google-containers@googlegroups.com
 
+# Getting help
+
+If your problem isn't answered by any of guides above, there are variety of ways for you to get help from the Kubernetes team.
+
+## Questions
+
+We have a number of FAQ pages
+   * [User FAQ](https://github.com/GoogleCloudPlatform/kubernetes/wiki/User-FAQ)
+   * [Debugging FAQ](https://github.com/GoogleCloudPlatform/kubernetes/wiki/Debugging-FAQ)
+   * [Services FAQ](https://github.com/GoogleCloudPlatform/kubernetes/wiki/Services-FAQ)
+
+You may also find the StackOverflow topics relevant
+   * [Kubernetes](http://stackoverflow.com/questions/tagged/kubernetes)
+   * [Google Container Engine - GKE](http://stackoverflow.com/questions/tagged/google-container-engine)
+
+## Bugs and Feature requests
+
+If you have what looks like a bug, or you would like to make a feature request, please use the [Github issue tracking system](https://github.com/GoogleCloudPlatform/kubernetes/issues).
+
+Before you file an issue, please search existing issues to see if your issue is already covered.
+
+# Help! My question isn't covered!  I need help now!
+
+## IRC
+
+The Kubernetes team hangs out on IRC at [```#google-containers```](https://botbot.me/freenode/google-containers/) on freenode.  Feel free to come and ask any and all questions there.
+
+## Mailing List
+
+The Kubernetes mailing list is google-containers@googlegroups.com
```

Comment:
maybe write this as
`[google-containers@googlegroups.com](https://groups.google.com/forum/#!forum/google-containers)